### PR TITLE
Added changes for notes listing UI #1

### PR DESCRIPTION
**Pull request by nivetha-yadhavan**
image

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -31,9 +21,7 @@ const DeleteAlert = ({
       onSubmit={handleDelete}
       onClose={onClose}
       message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      title={"Delete note?"}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:24]: No need for the braces here. [Refer](https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Rendering%20strings%20and%20blank%20spaces%20in%20JSX).

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <Dropdown buttonStyle="text" icon={MenuVertical} position="bottom-end">
+        <li>Edit</li>
+        <li>Delete</li>
+      </Dropdown>
+    </div>
+    <div className="mt-1 mb-3" style={{ color: "#68737D" }}>
+      <Typography style="body2">{note.desc}</Typography>
+    </div>
+    <hr></hr>
+
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:19]: No need to add blank lines in JSX. It's visually not helpful but rather makes it more confusing given that this is the presentation logic.

PS: I am talking about JSX here. Not JavaScript code where blank lines can make sense visually.

File: app/javascript/src/components/Dashboard/Notes/NotesMenu.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        <MenuBar.Block label="All" count={200} active />
+        <MenuBar.Block label="Users" count={80} />
+        <MenuBar.Block label="Leads" count={60} />
+        <MenuBar.Block label="Visitors" count={60} />
+
+        <MenuBar.SubTitle
+          iconProps={[
+            {
+              icon: Search,
+              onClick: () => setIsSearchCollapsed(!isSearchCollapsed),
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NotesMenu.jsx:22]: Please go through: https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 4]
```diff
@@ -1,102 +1,69 @@
...
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
 
+import { NOTES as notesList } from "./constants";
 import DeleteAlert from "./DeleteAlert";
+import Note from "./Note";
+import NotesMenu from "./NotesMenu";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:12]: https://academy.bigbinary.com/learn-react/pages/structuring-code

**Review by yedhink — COMMENTED** [review]
@nivetha-yadhavan I've added some comments.

- I have not pinpointed each and every spot a similar change ought to be made. Thus when you work on a review comment, please make sure to check your whole codebase to see where a similar change needs to be made.
- You can make these changes as part of your next feature PR or raise a separate PR just fixing all the review changes.

**Comment by nivetha-yadhavan** [comment]
@yedhink addressed the review comments in https://github.com/nivetha-yadhavan/wheel/pull/9

**Pull request merged**